<?xml version="1.0"?>

<launch>
	
	<arg name="ACCESS_URI" default="http://localhost:8442/serviceregistry/"/>
	<arg name="THIS_ADDRESS" default="130.240.5.68"/>
	<arg name="SERVICE_URI" default="Custom_url"/>
	<arg name="THIS_SYSTEM_NAME" default="temperature_subscriber_demo"/>
	<arg name="SERVICE_NAME" default="temperature_f"/>
	<arg name="INTERFACE" default="JSON"/>
	<arg name="UNIT" default="Fahrenheit"/>
	<arg name="SECURITY" default="Token"/>
	<arg name="THIS_PORT" default="8494"/>
	<arg name="SECURE_ARROWHEAD_INTERFACE" default="false"/>
	<arg name="SECURE_PROVIDER_INTERFACE" default="false"/>


	<!-- service -->
	<node pkg="subscriber" type="subscriber" name="subscriber">
		<param name="PRIVATE_KEY_PATH" value="Absolute_path_to_private_key"/>
		<param name="PUBLIC_KEY_PATH" value="Absolute_path_to_public_key"/>
		<param name="ACCESS_URI" value="$(arg ACCESS_URI)"/>
		<param name="THIS_ADDRESS" value="$(arg THIS_ADDRESS)"/>
		<param name="SERVICE_URI" value="$(arg SERVICE_URI)"/>
		<param name="THIS_SYSTEM_NAME" value="$(arg THIS_SYSTEM_NAME)"/>
		<param name="SERVICE_NAME" value="$(arg SERVICE_NAME)"/>
		<param name="INTERFACE" value="$(arg INTERFACE)"/>
		<param name="UNIT" value="$(arg UNIT)"/>
		<param name="SECURITY" value="$(arg SECURITY)"/>
		<param name="THIS_PORT" value="$(arg THIS_PORT)"/>
		<param name="SECURE_ARROWHEAD_INTERFACE" value="$(arg SECURE_ARROWHEAD_INTERFACE)"/>
		<param name="SECURE_PROVIDER_INTERFACE" value="$(arg SECURE_PROVIDER_INTERFACE)"/>
	</node>


</launch>

